applications/contrastive_phenotyping/contrastive_cli/predict.yml

```diff
@@ -0,0 +1,44 @@
+seed_everything: 42
+trainer:
+  accelerator: gpu
+  strategy: auto
+  devices: auto
+  num_nodes: 1
+  precision: 32-true
+  callbacks:
+    - class_path: viscy.light.embedding_writer.EmbeddingWriter
+      init_args:
+        output_path: 2024_02_04-tokenized-drop_path_0_0.zarr
+  inference_mode: true
```

Comment:
lightning_logs are written in current directory and not the parent of `output_path`. I suggest changing that behavior so that zarr stores with embeddings and lightning_logs are in the same folder.

Reply:
The log location is controlled separately by the logger callback, which assumes the default behavior here.